### PR TITLE
Import API basic support (files and non-sync'ed URLs)

### PR DESCRIPTION
**Pull request by danicarrion**

### CODE REVIEW

**Comment by danicarrion** [comment]
@javisantana please CR

File: README.md [hunk 1]
```diff
@@ -69,22 +65,49 @@ except CartoDBException as e:
...
+```
+
+Or just one:
+
+```python
+im = ImportManager(cl)
+single_import = im.get("afaab071-dc95-4bda-a772-ea37f8729157")
+```
+
+You can update the attributes in a import job any time (like for checking if an import has finished):
```
Comment (javisantana) [inline README.md]: how do I check it finish?

**Comment by javisantana** [comment]
looks pretty good, nice addiction

**Comment by danicarrion** [comment]
@javisantana now?

**Comment by javisantana** [comment]
:es:

**Pull request merged**